packages/documentation/src/stories/utilities/heading.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
+
+<Meta
+  title="Utilities/Heading"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+  }}
+/>
+
+# Headings
+
+<p className="lead">
+  Use the heading utilities, to match the font styling of a heading without the need of the
+  associated HTML element.
+</p>
+
+<div className="alert alert-warning">
+  <p>
+    We point out that semantic headings are important for{' '}
+    <a href="/?path=/docs/foundations-accessibility--page">accessibility</a>.
+  </p>
+  <p>Use these classes only in exceptional cases.</p>
+</div>
+
+## Usage on heading Elements
+
+Use the utility classes (`.h1` - `.h6`), to change the appereance of heading elements.
+
+<Canvas withSource="open">
+  <Story name="Heading Elements">
+    <h6 className="h3">Heading level 1 with h3 styles...</h6>
```

Comment:
```suggestion
    <h6 className="h3">Heading level 6 with h3 styles...</h6>
```

Reply:
But why would you change this? It is a <h6> tag.